UnityProject/App.config

```diff
@@ -25,6 +25,34 @@
         <assemblyIdentity name="System.Xml" publicKeyToken="b77a5c561934e089" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
remove these changes as well because I don't know what they are

Reply:
it done